src/templates/layout.js

```diff
@@ -68,234 +67,265 @@ const StyledLink = styled(PlainLink)`
-    <HorizontalRule width={1} />
-    <Flex
-      flexDirection={["column", "row"]}
-      justifyContent="space-between"
-      mt="3"
-    >
-      <Box width={[1, 1 / 3]}>
-        <P>
-          Join our <B>NEW</B> newsletter to learn about the latest trends in the
-          fast changing front end atmosphere
-        </P>
-        <Box my="1">
-          <form
-            action="https://codebrahma.us3.list-manage.com/subscribe/post"
-            method="post"
-            target="/thanks"
-          >
-            <input type="hidden" name="u" value="42069ce8928af0d4afc3fd428" />
-            <input type="hidden" name="id" value="a540ccf305" />
-            <InputText
-              name="MERGE0"
-              type="email"
-              placeholder="Email address"
-              backgroundColor="black.4"
-              borderWidth={0}
-              borderRadius={[3, "4px 0 0 4px"]}
-              placeholder="@ Email address"
-              width={[1, 'auto']}
-            />
-            <InputButton
-              value="Subscribe"
-              name="subscribe"
-              px={2}
-              py="10px"
-              mt={1}
-              borderRadius={[3, "0 4px 4px 0"]}
-            />
-          </form>
+const Footer = ({ images }) => {
+  return (
+    <Box marginTop="6" pb="6">
+      <Box height='2px' bg='black.1'/>
+      <Flex
+        flexDirection={["column", "row"]}
+        justifyContent="space-between"
+        mt="3"
+      >
+        <Box width={['100%', 1 / 3]}>
+          <P lineHeight='1'>
+            Join our <B>NEW</B> newsletter to learn about the latest trends in the
+            fast changing front end atmosphere
+          </P>
+          <Box my="1">
+            <form
+              action="https://codebrahma.us3.list-manage.com/subscribe/post"
+              method="post"
+              target="/thanks"
+            >
+              <input type="hidden" name="u" value="42069ce8928af0d4afc3fd428" />
+              <input type="hidden" name="id" value="a540ccf305" />
+              <Box display={['block', 'flex']} mt='3' align='stretch'>
+                <Input
+                  name="MERGE0"
+                  type="email"
+                  backgroundColor="black.4"
+                  borderWidth={0}
+                  borderRadius={[3, "6px 0 0 6px"]}
+                  placeholder="@ Email address"
+                  height='47px'
+                  p={1}
+                  fontFamily='body'
+                  lineHeight='3'
+                />
+                <Button
+                  bg='primary'
+                  color='tint'
+                  name="subscribe"
+                  px={3}
+                  py="10px"
+                  height='47px'
+                  lineHeight='0'
+                  borderRadius={[3, "0 4px 4px 0"]}
+                  border='none'
+                  fontFamily='body'
+                  mt={[1, 0]}
+                >
+                  Subscribe
```

Comment:
Form requires a Button with Submit Type.

Kindly check all the button types involved in other forms